qdata/ir.py

```diff
@@ -1,5 +1,32 @@
+"""This module contains the intermediate representation of the parsed Operator QASM,
+including the QASMProgram class"""
+
 from enum import Enum
 
+
+
+class QasmProgram:
```

Comment:
moved to be with the rest of the IR classes, it didn't really fit with the parser